subcommands/git/cmd_unix.go

```diff
@@ -0,0 +1,10 @@
+//go:build !windows
+// +build !windows
+
+package git
+
+import "golang.org/x/sys/unix"
+
+func isWritable(dir string) bool {
+	return unix.Access(dir, unix.W_OK) == nil
```

Comment:
I think this is the same as in my PR #206 where I use `syscall.Access(p, syscall.O_RDWR)` .